html/browsers/history/the-history-interface/history_go_to_uri-1.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<script src="history.js"></script>
+<script>
+  onunload = function() {}
+
+  onload = function() {
+    if (!opener.started) {
+      queue_next();
+    } else {
+      opener.pages.push(id);
+      opener.start_test_wait();
+      if (!opener.gone) {
+          history.go("history_entry.html");
```

Comment:
Not sure who can approve this PR, but is there anything else blocking the merge?